modules/gitops-release-notes-1-18-0.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-18-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-18-0_{context}"]
+= Release notes for {gitops-title} 1.18.0
+
+{gitops-title} 1.18.0 is available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, 4.18, and 4.19.
+
+[id="errata-updates-1-18-0_{context}"]
+== Errata updates
+
+RHEA-XXXX:NNNN - {gitops-title} 1.18.0 security update advisory::
+Issued: 2025-09-24
++
+The list of enhancements that are included in this release are documented in the following advisory:
++
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-18-0_{context}"]
+== New features
+
+[id="fixed-issues-1-18-0_{context}"]
+== Fixed issues
+
```

Comment:
Should this have a value such as "None"?

Reply:
Hey @briandooley, this section will be populated once I create the release notes PR for the GitOps 1.18 release. So I thought of creating this as a standalone now and using it in a later PR.